### ISSUE TITLE
Unable to do mup deploy as root user after 1.4.2

### ISSUE THREAD

**Issue by productiveme** [zodern/meteor-up#345]
I am trying to set up a CI server to run `mup deploy` as the root user, but I'm getting the following error when using meteor 1.4.2:

```
You are attempting to run Meteor as the "root" user. If you are developing,
this is almost certainly *not* what you want to do and will likely result in
incorrect file permissions. However, if you are running this in a build process
(CI, etc.) or you are absolutely sure you know what you are doing, add the
`--unsafe-perm` flag to this command to proceed.


=> Build Error. Check the logs printed above.
```

Could this flag perhaps be added to the mup command line?

**Comment by Deadlyweed** [comment]
@productiveme I got that a couple times too, I don't why it started do that but do this
 mup deploy --unsafe-perm
I even got it on creating a new app. So i don't think it has anything to do with mup

**Comment by productiveme** [comment]
@Deadlyweed I tried that but it seems the flag is not passed along to the meteor cli, since I get the same error.

**Comment by abernix** [comment]
For what it's worth, since the bug was fixed upstream in Meteor 1.4.2.1 and higher (meteor/meteor#7959), this should be working once again if you're using `abernix/meteord:base`.
